2.process.py

```diff
@@ -0,0 +1,108 @@
+import os
+import collections
+
+import pandas
+
+def create_history_df(path):
+    """
+    Process `gene_history.gz` for Project Cognoma. Returns a dataframe which
+    maps old GeneIDs to new GeneIDs. GeneIDs are not mapped to themselves, so
+    make sure not to filter all genes without an old GeneID when using this
+    dataset.
+    """
+    
+    renamer = collections.OrderedDict([
+        ('Discontinued_GeneID', 'old_entrez_gene_id'),
+        ('GeneID', 'new_entrez_gene_id'),
+        ('Discontinue_Date', 'date'),
+        ('#tax_id', 'tax_id'),
+    ])
+
+    history_df = (
+        pandas.read_table(path, compression='gzip', na_values='-')
+        [list(renamer)]
+        .rename(columns=renamer)
+        .query("tax_id == 9606")
+        .drop(['tax_id'], axis='columns')
+        .dropna(subset=['new_entrez_gene_id'])
+        .sort_values('old_entrez_gene_id')
+    )
+    history_df.new_entrez_gene_id = history_df.new_entrez_gene_id.astype(int)
+    
+    return history_df
+
+def create_gene_df(path):
+    """
+    Process `Homo_sapiens.gene_info.gz` for Project Cognoma. Filters genes to
+    homo sapiens with `tax_id == 9606` to remove Neanderthals et al.
+    """
+    
+    renamer = collections.OrderedDict([
+        ('GeneID', 'entrez_gene_id'),
+        ('Symbol', 'symbol'),
+        ('description', 'description'),
+        ('chromosome', 'chromosome'),
+        ('type_of_gene', 'gene_type'),
+        ('Synonyms', 'synonyms'),
+        ('Other_designations', 'aliases'),
+        ('#tax_id', 'tax_id'),
+    ])
+
+    gene_df = (pandas.read_table(path, compression='gzip', na_values='-')
```

Comment:
(specifically the pandas approach here - cleaner than CSV parsing by field index)